src/cloudflare/internal/ai-api.ts

```diff
@@ -98,9 +104,18 @@ export class Ai {
     /*
      * @deprecated this method is deprecated, do not use this
      */
-    public getLogs(): Array<string> {
+    public getLogs(): string[] {
       return []
     }
+
+    private async _parseError(res: Response): Promise<InferenceUpstreamError> {
+      try {
+        const content = (await res.json()) as AiError
+        return new InferenceUpstreamError(`${content.internalCode}: ${content.description}`, content.name);
+      } catch {
+        return new InferenceUpstreamError(await res.text());
+      }
+    }
```

Comment:
Will this work? If you call `.json()` and consume the response body, you can't then call `.text()` on it if that throws, can you?